### PR TITLE
8.0 pxc 2997

### PR DESCRIPTION
**Pull request by kennt-percona**

### CODE REVIEW

**Comment by dbpercona** [comment]
PXC-2997

File: scripts/wsrep_sst_common.sh [hunk 1]
```diff
@@ -648,31 +556,29 @@ function write_wsrep_state_file()
...
 #           2. shut down MySQL server
 #               Because the system tables use MyISAM (5.7 donor), the
 #               account mgmt commands (like CREATE USER, FLUSH PRIVILEGES)
 #               will not work. So the only way to use this new user is
 #               to restart the server.  Note that the SST user is not
 #               removed on shutdown, because the 'DROP USER' call will fail.
-#           3. start up MySQL Server (#2)
-#           4. run mysql_upgrade with the SST user
-#           5. shut down the MySQL server
+
```
Comment (dutow) [inline scripts/wsrep_sst_common.sh]: Missing `#` in this line

File: scripts/wsrep_sst_common.sh [hunk 2]
```diff
@@ -648,31 +556,29 @@ function write_wsrep_state_file()
...
 #               this time, the SST user will be removed on shutdown.
-#           6. start up MySQL Server (#3) (creates the SST user)
-#           7. run 'RESET SLAVE ALL' (if needed)
-#           8. shutdown the MySQL Server
+#           3. start up MySQL Server (#3) (creates the SST user)
+#           4. run 'RESET SLAVE ALL' (if needed)
+#           5. shutdown the MySQL Server
 #
-#   TODO: Steps (5) and (6) are needed due to 8.0.13 not restoring the
+#   TODO: Steps (2) and (3) are needed due to 8.0.13 not restoring the
```
Comment (dutow) [inline scripts/wsrep_sst_common.sh]: 8.0.18

**Review by dutow — APPROVED** [review]
Looks good, only two minor comment related comments

**Comment by kennt-percona** [comment]
https://pxc.cd.percona.com/job/pxc80-test-mtr/157/

File: scripts/wsrep_sst_common.sh (hunk 2 above)
Comment (kennt-percona) [reply scripts/wsrep_sst_common.sh]: ok (i actually removed the entire comment section since it's out of date)

**Pull request merged**